runtime/executor/src/main/java/org/apache/nemo/runtime/executor/data/PipeManagerWorker.java

```diff
@@ -189,10 +189,10 @@ public void onInputContext(final ByteInputContext inputContext) throws InvalidPr
 
   private int getNumOfPipeToWait(final RuntimeEdge runtimeEdge) {
     final int dstParallelism = ((StageEdge) runtimeEdge).getDstIRVertex().getPropertyValue(ParallelismProperty.class)
-      .orElseThrow(() -> new IllegalStateException());
```

Comment:
This issue was also assigned to another team. Please resolve [this issue](https://sonarcloud.io/project/issues?fileUuids=AWkltI3qgtFwgfOLu2CE&id=org.apache.nemo%3Anemo-project&resolved=false&rules=squid%3AS1192%2Csquid%3AS1602%2Csquid%3AS1612).

Reply:
Please revert the codes changed in this file too, so that no conflict occurs :)

Reply:
https://sonarcloud.io/project/issues?fileUuids=AWkltI3qgtFwgfOLu2B4&id=org.apache.nemo%3Anemo-project&resolved=false&rules=squid%3AS1192%2Csquid%3AS1602%2Csquid%3AS1612

Isn't this assigned to Team16?